worlds/sm64ex/docs/setup_en.md

```diff
@@ -59,6 +59,56 @@ If everything worked out, you will see a textbox informing you the connection ha
+
+IMPORTANT NOTE: The remainder of this section uses copy-and-paste code that assumes you're using the US version. If you instead use the Japanese version, you'll need to edit the EXE name accordingly by changing "sm64.us.f3dex2e.exe" to "sm64.jn.f3dex2e.exe".
+
+### Making an offline.bat for launching offline patch files
+
+Open Notepad. Paste in the following text: `start sm64.us.f3dex2e.exe --sm64ap_file %1 --skip-intro`
+
+Go to File > Save As...
+
+Navigate to the folder you selected for your SM64 build when you followed the Build guide for SM64PCLauncher earlier. Once there, navigate further into `build` and then `us_pc`. This folder should be the same folder that `sm64.us.f3dex2e.exe` resides in. 
+
+For "File name:" name it as: `"offline.bat"` . THE QUOTE MARKS ARE IMPORTANT! Otherwise it will create a text file instead ("offline.bat.txt") which won't work as a batch file until you alter the extension properly.
+
+Now you should have a "offline.bat" file with a gear icon in the same folder as your "sm64.us.f3dex2e.exe". (If the icon is instead a notepad, you saved it as a TXT file instead and need to change the extension to BAT). Right click that offline.bat file and choose Send To > Desktop (Create Shortcut).
+
+The batch file you just created and made a shortcut for is set up to accept patch files dragged into it. Start an offline singleplayer game if you haven't already, download the `.apsm64ex` patch file that was prepared by the seed generation, and drag-n-drop that onto the batch file to open the game and start playing.
+
+NOTE: When playing offline patch files, a `.save` file is created in the same directory as your patch file, which contains your save data for that seed. Don't delete it until you're done with that seed.
+
+### Making an online.bat for launching online Multiworld games
+
+The steps here are very similar, as you will be making a batch file in the same location as the offline one. However, the text you put into the batch file is different, and you won't be dragging patch files onto it.
+
+Use the same steps as before to open Notepad and paste in the following:
+
+`set /p port="Enter port number of room - "`
+
+`set /p slot="Enter slot name - "`
+
+`start sm64.us.f3dex2e.exe --sm64ap_name "%slot%" --sm64ap_ip archipelago.gg:%port% --skip-intro`
+
+Save and shortcut the same as before. Unlike the offline batch file, you'll open this one simply by double-clicking it. The window that opens will ask you for the port of the room you wish to join - this five-digit number will be provided by the seed page the game host will share with all players post-generation. Then it will ask you for your slot name - this name is set up in your YAML, and also will be shown on the same seed page previously mentioned, if you need to double-check it.
```

Comment:
```suggestion
Save this file as `"online.bat"`, then create a shortcut by following the same steps as before. 

To use this batch file,  double-click it. A window will open. Type the five-digit port number of the room you wish to join, then type your slot name.
- The port number is provided on the room page. The game host should share this page with all players.
- The slot name is whatever you typed in the "Name" field when creating a config file. All slot names are visible on the room page.
```
Just a couple of reformatting suggestions. Also, I'm leading the charge to remove "YAML" from our setup guides.